docs/getting_started/index.md

```diff
@@ -13,9 +13,9 @@
 
 ## Dependencies needed for installation
 
--   [Python 3](https://www.python.org/downloads/)
+-   [Python 3](https://www.python.org/downloads/). After installation, check that Python has been added to path in environment variables. And if it hasn't been added, see [how to add it here](https://realpython.com/add-python-to-path/).
 -   **libjpeg** and **zlib**, libraries required for Django's **Pillow** library.
-    See Pillow's [platform-specific installation instructions](https://pillow.readthedocs.io/en/stable/installation.html#external-libraries).
+    See Pillow's [platform-specific installation instructions](https://pillow.readthedocs.io/en/stable/installation.html#external-libraries). During the installation, if `python3` doesn't work, you can be replace it with `py`.
```

Comment:
I recommend we add a 'common problems' or 'troubleshooting' section at the end and put these notes there. Otherwise this list will become too hard to read. Also avoid URLs where the link is like 'go here'.


```md
... at the end before the TOC
-   [Feature Detection](image_feature_detection)

(common_installation_issues)=

## Common issues

### Python is not available in `path`

```sh
python
> command not found: python
```

For detailed guidance see this guide on [how to add Python to your path](https://realpython.com/add-python-to-path/).

### `python3` not available

```sh
python3 -m pip install --upgrade pip
> command not found: python3
```

Something here about using `py`.

```{toctree}
---
maxdepth: 1
---
tutorial
demo_site
integrating_into_django
the_zen_of_wagtail
```

```